specsim/config.py

```diff
@@ -370,7 +374,7 @@ def load_table(self, parent, column_names, interpolate=True, as_dict=False):
             # Look for parent.table.path first.
             paths.append(os.path.join(self.abs_base_path, node.path))
         except AttributeError:
-            path_keys = node.paths.keys()
+            path_keys = list(node.paths.keys())
```

Comment:
@weaverba137 How does your comment apply here?  There is no iteration here.  Also, this class implements a custom keys() method.